site.yml

```diff
@@ -1,5 +1,6 @@
 site:
   title: ownCloud Documentation
+  url: http://localhost:5000
```

Comment:
Hardcoded to localhost doesn't look good.

Reply:
True. I got stuck here and need to fix this.

Reply:
I changed this to something akin to a Docker environment variable.